common/djangoapps/student/views.py

```diff
@@ -318,6 +318,30 @@ def get_course_enrollments(user, orgs_to_include, orgs_to_exclude):
             yield enrollment
 
 
+def get_course_certificates(user):
+    """
+    Returns information about a user's course certificates.
+    """
+    # Let's filter out any courses in an "org" that has been declared to be
+    # in a configuration
+    org_filter_out_set = configuration_helpers.get_all_orgs()
```

Comment:
This code was confusing so I didn't get far on a review of this code.  Instead, I worked on a refactor that is untested, but I hope works. :)
https://github.com/edx/edx-platform/pull/15899/files